drivers/wifi/nxp/nxp_wifi_drv.c

```diff
@@ -1197,18 +1197,23 @@ static int nxp_wifi_power_save(const struct device *dev, struct wifi_ps_params *
 			}
 			break;
 		case WIFI_PS_PARAM_MODE:
+			
 			if (params->mode == WIFI_PS_MODE_WMM) {
+				#if CONFIG_WMM_UAPSD
```

Comment:
Please use:
#ifdef CONFIG_NXP_WIFI_WMM_UAPSD

Reply:
Sure, I will make the changes according to your request.
There is a modification that I need your help with:
Regarding `hal_nxp/blob/master/zephyr/module.yml` , I don't know how to add `sduart8987_wlan_bt.bin.inc` or other binary files needed in `hal/nxp/mcux/middleware/wifi_nxp/wifi_bt_firmware`. Could you please help add them to `module.yml` ?
https://github.com/zephyrproject-rtos/hal_nxp/blob/master/zephyr/module.yml#L19

Reply:
There is another question I would like to ask, also regarding the `modules/hal/nxp/zephyr/module.yml` file:
From the code in `samples/net/wifi/shell/sample.yaml`, it seems that `samples/net/wifi/shell` currently supports `frdm_rw612` and `rd_rw612_bga`.
I used the following commands to compile:
```
west update
```

```
west build -b frdm_rw612 zephyr/samples/net/wifi/shell/
```
or
```
west build -b rd_rw612_bga zephyr/samples/net/wifi/shell/
```

Both commands result in the following error:
```
CMake Error at /home/csbobo/code/zephyr_code/modules/hal/nxp/zephyr/src/rw61x/CMakeLists.txt:45 (message):
  Couldn't find signed firmware!
  /home/csbobo/code/zephyr_code/modules/hal/nxp/zephyr/blobs/rw61x/rw61x_sb_wifi_a2.bin
```

In `modules/hal/nxp/zephyr/module.yml`, there is already a description related to `rw61x_sb_wifi_a2.bin`. What command can be used to obtain `rw61x_sb_wifi_a2.bin` and allow `frdm_rw612` and `rd_rw612_bga` to compile normally?

Reply:
Please use cmd:
`west blobs fetch hal_nxp`